recipe/meta.yaml

```
@@ -16,15 +15,15 @@ source:
 build:
   merge_build_host: true  # [win]
   number: 0
```

Please bump the version number, as e.g. the license spdx update is for all platforms
```suggestion
  number: 1
```